sqlpad.yaml

```diff
@@ -35,8 +35,8 @@ pipeline:
 
   - runs: |
       ./scripts/build.sh
-      mkdir -p ${{targets.destdir}}/usr/bin
-      mv server ${{targets.destdir}}/usr/bin/sqlpad-server
+      mkdir -p "${{targets.destdir}}/usr/app"
```

Comment:
This is not a standard FHS path, so if you need this for compatibility, it needs to be in a `-compat` package.

Reply:
Just tried to fix this but I am not able to run `node server.js --config ./config.dev.env` after installing this package inside /usr/app/sqlpad-server, it says:

```
[sdk] ❯ /usr/app/sqlpad-server node server.js --config ./config.dev.env
bash: /usr/app/sqlpad-server: No such file or directory
[sdk] ❯ cd /usr/app/
[sdk] ❯ ls
sqlpad-server
```

I am not sure if it is expected because of softlink or we have to find better way

Reply:
Just fixed, thanks for the review